### PR TITLE
Several template improvements

### PR DESCRIPTION
**Pull request by emanuelschuetze**
- Move Languages services from site.js to base.js
  Now the detected browser language is used as projector language.
- Use verboseName instead of agendaSupplement.
- Show submitters on projector (load Users on Controller).
- Improve list of speakers (slide and item detail).

### CODE REVIEW

File: openslides/agenda/static/templates/agenda/item-detail.html [hunk 1]
```diff
@@ -22,20 +22,25 @@
...
         <i class="fa fa-video-camera"></i>
+        {{ item.getContentResource().verboseName }}
       </a>
     </div>
     <h1>{{ item.getTitle() }}</h1>
-    <h2><translate>List of speakers</translate></h2>
+    <h2>
+      <translate>List of speakers</translate>
+      <span ng-if="item.speaker_list_closed" class="slimlabel label label-danger"
+          translate>Closed</span>
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/item-detail.html]: Please try to find a way not to add the word closed in the title h2

File: openslides/agenda/static/templates/agenda/slide-list-of-speakers.html [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
-  <p>
-    <strong ng-repeat="speaker in item.speakers | filter: {end_time: null, begin_time: '!!'}">
+  <!-- Title -->
+  <div id="title">
+    <h1>{{ item.getTitle() }}</h1>
+    <h2>
+      <span translate>List of speakers</span>
+      <span ng-if="(item.speakers | filter: {begin_time: null}).length > 0">
+      &ndash; {{ (item.speakers | filter: {begin_time: null}).length }} <translate>speakers</translate>
+      <span ng-if="item.speaker_list_closed" class="slimlabel label label-danger" translate>Closed</span>
```
Comment (ostcar) [inline openslides/agenda/static/templates/agenda/slide-list-of-speakers.html]: Here the same

File: openslides/agenda/static/templates/agenda/item-detail.html (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/agenda/static/templates/agenda/item-detail.html]: No. That's exactly what I want. See also http://getbootstrap.com/components/#labels

**Comment by ostcar** [comment]
@emanuelschuetze Please fix the notes in another pull request

**Pull request merged**